### PR TITLE
PLANET-2318 - Move out plugin-blocks frontend code

### PR DESCRIPTION
**Pull request by comzeradd**
This removes all css/js code related to blocks and moves them to plugin-blocks.

greenpeace/planet4-plugin-blocks#301

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -418,19 +418,26 @@ public function enqueue_admin_assets( $hook ) {
...
+		// plugin-blocks assets.
+		$css_blocks_creation = filectime( get_template_directory() . '/blocks.css' );
+		$js_blocks_creation  = filectime( get_template_directory() . '/assets/js/blocks.js' );
+
 		// CSS files
 		wp_enqueue_style( 'bootstrap', 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.1/css/bootstrap.min.css', array(), '4.1.1' );
 		wp_enqueue_style( 'slick', 'https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.css', array(), '1.9.0' );
 		wp_enqueue_style( 'font-awesome', $this->theme_dir . '/assets/lib/@fortawesome/fontawesome-free-webfonts/css/fontawesome.css', array(), '5.0.10' );
 		wp_enqueue_style( 'parent-style', $this->theme_dir . '/style.css', [], $css_creation );
+		wp_enqueue_style( 'plugin-blocks', $this->theme_dir . '/blocks.css', [], $css_blocks_creation );
```
Comment (kirdia) [inline functions.php]: @comzeradd Can we move the enqueueing of blocks style and scripts to the plugin itself or it creates problems?
Then we can change planet4-base to generate the final files inside plugins folder.

Comment (angtheod) [reply functions.php]: Agreed. If its possible we should go for it, in favor of modulatiry

Comment (angtheod) [reply functions.php]: @comzeradd any update on this? Can we move the style/scripts inside the blocks plugin directory?

Comment (comzeradd) [reply functions.php]: @angtheod another commit was just added by @kirdia if you could have a final look before we merge it.

**Pull request merged**